### PR TITLE
ProductAnalysis

### PR DESCRIPTION
**Pull request by maximuschoi-uiuc**

### CODE REVIEW

File: ProductAnalysis.py [hunk 1]
```diff
@@ -21,24 +21,74 @@
...
+  for r in range(len(reviewDataList)):
+    for review in reviewDataList[r]:#['Reviews']:
+      parsedWords = parseWords(review['fullText'], stopWords)
+      reviewFrequency = dict(nltk.FreqDist(parsedWords))
+      reviewFreqDictList.append(reviewFrequency)
+      reviewList.append(parsedWords)
+      reviewIdList.append(review['reviewId'])
+      productIdList.append(productList[r])
+      reviewContentList.append(review['fullText'])
+      allTerms += parsedWords
```
Comment (jackdedobb) [inline ProductAnalysis.py:42]: should you be adding the len(parsedWords) or should it be adding it to a set of all the word terms and allTerms should be implemented as a set instead?

File: ProductAnalysis.py [hunk 2]
```diff
@@ -21,24 +21,74 @@
...
+  termFrequency = nltk.FreqDist(allTerms)
+  vocab, cnt = [], []
+  vocabDict = {}
+  for k,v in termFrequency.items():
+    if v > 5:
```
Comment (jackdedobb) [inline ProductAnalysis.py:47]: wheres magic number 5 come from?

File: ProductData/CleanData/B00004SDFG.json [hunk 3]
```diff
@@ -1 +1 @@
...
...options played friends mp3 player sound quality highly recommend product spends lot time", "rating": 5.0, "helpfulNum": 32, "totalNum": 33, "commentNum": 0, "webUrl": "http://www.amazon.com/review/product/B00004SDFG/ref=cm_cr_dp_all_summary/002-2313684-7157658?%5Fencoding=UTF8&showViewpoints=1&sortBy=bySubmissionDateAscending", "htmlPath": "F:\\amazon_mp3\\1/854"}, {"reviewId": "54525", "productName": "i2Go eGo 32MB MP3 Player (Blue)", "title": "Best MP3 Player for your Money!", "author": "M. Phillips", "createDate": "2000-07-13 00:00:00", "fullText": "overall player packs punch player compete storage capacity buy additional storage 32mb included innovative features using website customize own news sports weather content downloading load ibm microdrive whopping 340mb storage space buy optional compact flash expansion slot locked amount memory add flash card technology improves perfect car product worth money", "rating": 5.0, "helpfulNum": 19, "totalNum": 24, "commentNum": 0, "webUrl": "http://www.amazon.com/review/product/B00004SDFG/ref=cm_cr_dp_all_summary/002-2313684-7157658?%5Fencoding=UTF8&showViewpoints=1&sortBy=bySubmissionDateAscending", "htmlPath": "F:\\amazon_mp3\\1/854"}]
```
Comment (jackdedobb) [inline ProductData/CleanData/B00004SDFG.json]: what was done to all the clean data files?

**Review by jackdedobb — CHANGES_REQUESTED** [review]

File: ProductData/CleanData/B00004SDFG.json (hunk 3 above)
Comment (jackdedobb) [reply ProductData/CleanData/B00004SDFG.json]: done.....resolved

File: ProductAnalysis.py (hunk 1 above)
Comment (maximuschoi-uiuc) [reply ProductAnalysis.py:42]: I kept it consistent with HotelAnalysis. @ssubramanian90 can yo help answer Jackson's questions?

File: ProductAnalysis.py (hunk 2 above)
Comment (maximuschoi-uiuc) [reply ProductAnalysis.py:47]: I kept it consistent with HotelAnalysis. @ssubramanian90 can yo help answer Jackson's questions?

File: ProductAnalysis.py (hunk 1 above)
Comment (ssubramanian90) [reply ProductAnalysis.py:42]: Lets' keep it consistent for now to see if everything is working and then improve the code.

**Review by ssubramanian90 — APPROVED** [review]

**Pull request merged**